tasks/src/feature-extraction/data.ts

```diff
@@ -0,0 +1,51 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "Wikipedia dataset containing cleaned articles of all languages. The datasets are built from the Wikipedia dump (https://dumps.wikimedia.org/) with one split per language. Each example contains the content of one full Wikipedia article with cleaning to strip markdown and unwanted sections (references, etc.).",
+			id:"wikipedia",
+		}
+	],
+	demo: {
+		inputs: [
+			{
+				label:   "Input",
+				content:
+						"India, officially the Republic of India, is a country in South Asia.",
+				type: "text",
+			},
+
+		],
+		outputs: [
+			{
+				table:
+						["array of shape (1,m,768) where m is the number of tokens in the input."][""],
```

Comment:
Hmm, I don't know which way to represent this properly, pinging @osanseviero

Reply:
Hmm maybe the table could have some dummy values, and the label could explain what it contains? WDYT?

Reply:
@akshay-babbar how does this sound?

Reply:
Sorry i didn't see this and have added the entire 768 dim array to data.ts. Can you just have a look?
If the rendering of data.ts comes correct, i can replace it with a smaller dummy array or as you suggest.